pkg/northbound/gnmi/subscribe.go

```diff
@@ -129,38 +129,35 @@ func broadcastNotification() {
 		log.Println("Error while subscribing to updates", err)
 	}
 	for update := range changes {
```

Comment:
I think it would be more readable if "changes" were "changesChan". I think all channel variables should end in Chan, so you know what you're dealing with

Reply:
done